qa/suites/rgw/notifications/valgrind.yaml

```diff
@@ -0,0 +1,9 @@
+overrides:
+  install:
+    ceph:
+      #debuginfo: true
+  rgw:
+    client.0:
+      valgrind: [--tool=memcheck, --max-threads=1024] # http://tracker.ceph.com/issues/25214
+    client.1:
```

Comment:
i think that only the "others" task is using 2 RGWs. amqp and kafka have only "client.0"